src/Idris/REPL/Commands.hs

```diff
@@ -17,6 +17,7 @@ data Command = Quit
              | Reload
              | Watch
              | Load FilePath (Maybe Int) -- up to maximum line number
+             | RunShellCommand FilePath
```

Comment:
Should it not take a list of strings (because the command might take arguments)?